docs/operating-modes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Operating modes
+sidebar_position: 25
+custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/operating-modes.md
+---
+# Operating modes
+
+There are currently two different ways to run Stryker:
+
+- From the test project context
+- From the source project context
+
```

Comment:
Wait... you can run Stryker from wherever, if you specify both source and test projects, at least it works with full paths.

Reply:
Wait does that work?... I never tried it but I would expect stryker to fail if you run it from a location outside of your project

Reply:
It absolutely does :)

Reply:
Well it's been there for a while, we have 1.5.3 in our pipeline currently and we are running things like this. 
So I don't know why and since when it works but it does... might be worth checking that the new custom output option is okay with that and also adding some integration tests then.

Reply:
I see, I don't know if it's a good idea to add this to the documentation as it's not really officially supported right now. It can be added later when this feature is properly tested.